test/test_helper.rb

```diff
@@ -39,20 +39,38 @@ def enrollment_file
   }
 end
 
-def statewide_file
-end
-
 def enrollment_statewide_files
   {
     enrollment: {
-      kindergarten: "./test/sample_kindergarten.csv"
+      kindergarten: "./test/sample_kindergarten.csv",
+      high_school_graduation: "./test/sample_high_school.csv"
     },
     statewide_testing: {
-      third_grade: "./test/sample_third_grade.csv"
+      third_grade: "./test/sample_third_grade.csv",
+      eighth_grade: "./test/sample_eighth_grade.csv",
+      math: "./test/sample_math.csv",
+      reading: "./test/sample_reading.csv",
+      writing: "./test/sample_writing.csv"
     }
   }
 end
 
+def headcount_analyst_files
+  {
+  enrollment: {
+    kindergarten: "./data/Kindergartners in full-day program.csv",
+    high_school_graduation: "./data/High school graduation rates.csv",
+  },
+  statewide_testing: {
+    third_grade: "./data/3rd grade students scoring proficient or above on the CSAP_TCAP.csv",
+    eighth_grade: "./data/8th grade students scoring proficient or above on the CSAP_TCAP.csv",
+    math: "./data/Average proficiency on the CSAP_TCAP by race_ethnicity_ Math.csv",
+    reading: "./data/Average proficiency on the CSAP_TCAP by race_ethnicity_ Reading.csv",
+    writing: "./data/Average proficiency on the CSAP_TCAP by race_ethnicity_ Writing.csv"
+  }
+}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.